client/src/app/gateways/repositories/structure-levels/structure-level.action.ts

```diff
@@ -0,0 +1,5 @@
+export class StructureLevelAction {
+    public static readonly CREATE = `structure-level.create`;
+    public static readonly UPDATE = `structure-level.update`;
+    public static readonly DELETE = `structure-level.delete`;
```

Comment:
These are supposed to contain the backend action name, those should use `_`, not `-`

Reply:
I replaced `-` with `_` in the backend action names.